gatsby-node.js

```diff
@@ -9,45 +9,126 @@ const report = require('gatsby-cli/lib/reporter');
  */
 const identity = obj => obj;
 
+/**
+ * Fetches all records for the current index from Algolia
+ *
+ * @param {AlgoliaIndex} index eg. client.initIndex('your_index_name');
+ * @param {Array<String>} attributesToRetrieve eg. ['modified', 'slug']
+ */
+function fetchAlgoliaObjects(index, attributesToRetrieve) {
+  return new Promise((resolve, reject) => {
+    const browser = index.browseAll('', { attributesToRetrieve: ['modified'] });
```

Comment:
Why don't use `attributesToRetrieve`? It seems a bug.

```suggestion
    const browser = index.browseAll('', { attributesToRetrieve });
```

Reply:
Thanks, it is actually meant to be the fallback value. Have updated the code.